week06_style_transfer/style_transfer_pytorch.ipynb

```diff
@@ -26,6 +26,18 @@
     "# !pip3 install --user torchvision"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    def image_loader(image_name):
</pre><p>test 11 on line 12</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/Practical_DL/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>reply 1 to normal review comment</p>